account_payment_order/models/account_move.py

```diff
@@ -39,6 +39,21 @@ def _compute_payment_order_ok(self):
                 payment_mode = move.payment_mode_id
             move.payment_order_ok = payment_mode.payment_order_ok
 
+    def _get_communication(self):
```

Comment:
Be more specific on the name, as the method is located in account.move:

```suggestion
    def _get_payment_order_communication(self):
```